docs/src/terminology.md

```diff
@@ -11,39 +11,23 @@ Depth = 3
 
 ### Feature
 
-A quality or quantity associated with an atom that we wish to encode, such as atomic mass, row in the periodic table, etc.
+A quality or a quantity that is associated with an atom, which is of interest to us. A feature could be anything, like atomic mass, the row in the periodic table to which the element belongs, etc.
```

Comment:
can we add one or two more examples here to cover things like pair and environment features? Maybe something like

> A feature could be a property of an individual atom, such as its atomic mass or the row in the periodic table to which it belongs; or it could be a property of a pair of atoms (e.g. distance between them, presence of a bond), or even a fingerprint of an entire structure.

Reply:
Resolved in 02a9b4f.